src/utils/simplified-account/settings/SettingsBuilder.class.js

```diff
@@ -22,6 +22,7 @@ module.exports = class SettingsBuilder {
       throw new Error('Cannot add setting without category, use .category(name) first.')
     }
 
+    const urlParts = ['simplified', 'settings', `${id}`]
```

Comment:
`/${id}` might be slightly safer just in case one setting id is a substring of another

Reply:
done